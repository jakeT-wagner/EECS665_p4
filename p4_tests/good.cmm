int a;
int b;
string str;
bool c;
int v;

void fn() {
	write a;
	write b;
	read a;
	return a + b;
}

int cc(int n1, int n2) {
	if (c) {
		if (n1 >= n2){
			string str2;
			while (str2 != "2") {
				string n1;
				c = 3;
				return n1;
			}
		}
		else {
			string aaa;
		}
	}
}

int ccc(bool b2) {
	int hhhhh;
}

void main() {
	int n1;
	int n2;
	n2 = a + b;
	n1 = b; 
	cc(n1, n2);
}