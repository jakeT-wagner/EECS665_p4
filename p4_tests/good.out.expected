int a;
int b;
string str;
bool c;
int v;
void fn(){
	write a(int);
	write b(int);
	read a(int);
	return a(int) + b(int);
}
int cc(int n1, int n2){
	if (c(bool)){
		if (n1(int) >= n2(int)){
			string str2;
			while (str2(string) != "2"){
				c(bool) = 3;
			}
		} else {
			string aaa;
		}
	}
}
int ccc(bool b2){
	int hhhhh;
}
void main(){
	int n1;
	int n2;
	n2(int) = (a(int) + b(int));
	n1(int) = b(int);
	cc(int,int->int)(n1(int), n2(int));
}